modules/modules.nix

```diff
@@ -189,6 +189,7 @@ let
     (loadModule ./services/xscreensaver.nix { })
     (loadModule ./services/xsuspender.nix { condition = hostPlatform.isLinux; })
     (loadModule ./systemd.nix { })
+    (loadModule ./targets/darwin.nix { condition = hostPlatform.isDarwin; })
```

Comment:
I was forced to comment this line out in order to get home-manager working in conjunction with my nix-darwin installation again. I don't manage either one through the other, but both separately, and this caused a conflict over management of `~/Applications`.